### PR TITLE
[PUSH-139] - Update Gemfile

### PR DESCRIPTION
**Pull request by ndurell**
This should address most security alerts for this repo.

TZINFO (now at 2.0): https://github.com/klaviyo/klaviyo-swift-sdk/pull/30
cocoapods-downloader (now at 1.6.3): https://github.com/klaviyo/klaviyo-swift-sdk/security/dependabot/5
addressable (now at 2.8.3): https://github.com/klaviyo/klaviyo-swift-sdk/security/dependabot/3
json (now at 2.6.2): https://github.com/klaviyo/klaviyo-swift-sdk/security/dependabot/2
rubyzip (now at 2.3.2): https://github.com/klaviyo/klaviyo-swift-sdk/security/dependabot/1

# Test Plan
Test out commands in Fastfile.

### CODE REVIEW

File: fastlane/report.xml [hunk 1]
```diff
@@ -5,42 +5,14 @@
...
...endency order&#10;    - NOTE  | [iOS] xcodebuild:  warning: The macOS deployment target 'MACOSX_DEPLOYMENT_TARGET' is set to 12.5, but the range of supported deployment target versions is 10.9 to 12.3.99.&#10;    - WARN  | xcodebuild:  /Users/noah.durell/code/klaviyo-swift-sdk/Sources/KlaviyoSwift/Klaviyo.swift:501:29: warning: 'archiveRootObject(_:toFile:)' was deprecated in iOS 12.0: Use +archivedDataWithRootObject:requiringSecureCoding:error: and -writeToURL:options:error: instead&#10;    - WARN  | xcodebuild:  /Users/noah.durell/code/klaviyo-swift-sdk/Sources/KlaviyoSwift/Klaviyo.swift:511:29: warning: 'archiveRootObject(_:toFile:)' was deprecated in iOS 12.0: Use +archivedDataWithRootObject:requiringSecureCoding:error: and -writeToURL:options:error: instead&#10;    - WARN  | xcodebuild:  /Users/noah.durell/code/klaviyo-swift-sdk/Sources/KlaviyoSwift/Klaviyo.swift:542:45: warning: 'unarchiveObject(withFile:)' was deprecated in iOS 12.0: Use +unarchivedObjectOfClass:fromData:error: instead&#10;&#10;[!] KlaviyoSwift did not pass validation, due to 3 warnings (but you can use `--allow-warnings` to ignore them).&#10;You can use the `--no-clean` option to inspect any issue.&#10;" />
```
Comment (jordan-griffin) [inline fastlane/report.xml:15]: Do we want to deal with this failure message in the report before we merge this in?

Comment (ndurell) [reply fastlane/report.xml:15]: I dealt with that warning in another PR. So it should go away once that's merged but fair maybe I don't need to commit this? 🤔

Comment (jordan-griffin) [reply fastlane/report.xml:15]: Ya I think its fine to leave this out. Doesn't exactly seem critical to anything

**Review by jordan-griffin — APPROVED** [review]
approving but may want to put that `report.xml` in a gitignore or something

**Pull request merged**